Gemfile

```diff
@@ -80,6 +80,11 @@ group :test do
   #  gem "delorean"
 end
 
+# For heroku
+group :production do
+  gem 'pg'
+  gem 'rails_12factor'
+end
```

Comment:
This is somewhat making postgres the default db for tsp. I don't know how other projects solve this, but maybe it makes sense to have a heroku group here? Have to think about this...